### PR TITLE
Define and activate release v12.2.0 for AWS

### PR DESCRIPTION
**Pull request by alex-dabija**
AWS v12.2.0 release updates kiam and aws-cni.

### CODE REVIEW

**Comment by calvix** [comment]
KIAM should be `1.4.2` https://github.com/giantswarm/kiam-app/releases/tag/v1.4.2

**Comment by alex-dabija** [comment]
@calvix I just opened the initial PR, no changes. We now need to update the components (kiam, aws-cni, etc. ).

**Comment by alex-dabija** [comment]
The PR now includes kiam-app v1.4.2 .

File: aws/v12.2.0/release.diff [hunk 1]
```diff
@@ -0,0 +1,66 @@
+apiVersion: release.giantswarm.io/v1alpha1                         apiVersion: release.giantswarm.io/v1alpha1
+kind: Release                                                      kind: Release
+metadata:                                                          metadata:
+  annotations:                                                       annotations:
+    giantswarm.io/docs: https://docs.giantswarm.io/reference/cp        giantswarm.io/docs: https://docs.giantswarm.io/reference/cp
+  creationTimestamp: null                                            creationTimestamp: null
+  name: v12.1.3                                                 |    name: v12.2.0
```
Comment (sslavic) [inline aws/v12.2.0/release.diff:7]: 12.1.4 has been merged. Would you like me to updated this release definition or prefer to do it yourself?

Please don't downgrade external-dns to 1.3.0

Comment (alex-dabija) [reply aws/v12.2.0/release.diff]: I was aware of AWS v12.1.4 and I just merged master, updated release.yaml and regenerated release.diff to take into account v12.1.4.

**Comment by njuettner** [comment]
Needs to be in the release

aws-cni v1.7.2
aws-operator v9.0.0
cluster-operator v3.2.0
kiam v1.5.0


Potentially, can be updated:

cert-exporter v1.2.4
calico v3.16.1
cert-manager v1.0.1
coredns v1.7.0
etcd v3.4.13
kubernetes v1.17.11
metrics-server v0.3.7
net-exporter v1.9.2

**Comment by njuettner** [comment]
What we agreed on including in the release:

aws-cni v1.7.2
aws-operator v9.0.0
cluster-operator v3.2.0
kiam v1.5.0

calico v3.16.1
etcd v3.4.13
kubernetes v1.17.11

**Comment by njuettner** [comment]
Also we need to bump containerlinux, I'll update it as well, see validation errors:

```
--- FAIL: Test_Releases (0.22s)
    --- FAIL: Test_Releases/case_1:_aws_releases_are_valid (0.18s)
        releases_test.go:376: Release v12.2.0 does not meet the requested version requirements:
            requested: containerlinux: >= 2512.4.0 	actual: 2512.2.1
FAIL
FAIL	github.com/giantswarm/releases	0.254s
FAIL

**Comment by njuettner** [comment]
Tested upgrade v11.5-> v12.2 and created cluster v12 in China, looks good from my side

**Comment by njuettner** [comment]
tested more in gauss incl. upgrades, lgtm 🙂

**Review by calvix — APPROVED** [review]

**Review by njuettner — APPROVED** [review]

**Pull request merged**